examples/LCOMBS/testing.ipynb

```diff
@@ -287,7 +287,7 @@
     "\n",
     "visualizer.fit(X, y)                # Fit the data to the visualizer\n",
     "visualizer.transform(X)             # Transform the data\n",
-    "visualizer.poof()    # Draw/show/poof the data"
+    "visualizer.show()    # Draw/show/show the data"
    ]
```

Comment:
I just did a find and replace in the user example notebooks - I didn't want to mess with them too much since they are not maintained by us, but wanted to make sure they continued to run if users came back to them.